mock-core-configs/mock-core-configs.spec

```diff
@@ -124,8 +130,6 @@ else
     echo "Warning: file %{_sysconfdir}/mock/$cfg does not exist."
     echo "         unable to update %{_sysconfdir}/mock/default.cfg"
 fi
-:
```

Comment:
I want to definitely preserve this. In the past, this has been removed for a brief period and it caused a big pain for people where this script failed. For various reasons.

Reply:
Then it leaves a broken symlink in place. Wouldn't that, in fact, be a problem that should trigger a failure? Silent failures are *nasty*. I can restore that, make it a separate patch to explain better why we would choose to ignore a clear sscripting failure.

Reply:
Not able to finish rpm transaction is a much bigger problem for most people. You can search fedora-devel archives and closed bugzillas for the wave of reaction when this was fatal.
Problem is that when this is fatal, users are blocked until I release a new version of rpm package, which will take - at least - days.

Reply:
Yes, not only it is a bigger problem - it is also given by guidelines [1], search for *All scriptlets MUST exit with the zero exit status. ...*

[1] https://docs.fedoraproject.org/en-US/packaging-guidelines/Scriptlets/#_syntax

Reply:
It's a suggestion, not a mandate, and depends on other suggestions. In this case the "ln -sf" command has all its output piped to /dev/null, which results in failures with no visible error report. That's fairly nasty, and probably worthy of a distinct patch.

I'll put back the ":" because it does have a point I'd not realized, and that's not what the patch  was about.was supposed to be about.